### DISCUSSION TITLE
k8s operator

### DISCUSSION THREAD

**Discussion by kotyara85** [ory/hydra#3127] [Q&A] [answered]
Hello,
Are they any plans to implement k8s operator for hydra?
Thanks

**Comment by aeneasr** [comment, marked as answer]
Have you seen http://github.com/ory/hydra-maester ?

**Reply by kotyara85** [reply]
Will check it out, ty

**Comment by gdoteof** [comment]
hydra-maester is not a hydra operator.  It would be a useful part of an operator, but it looks like maester is just a controlelr for client requests, it doesn't handle the deployment of hydra itself.

there is a charmed operator for hydra.  i have tried to get that working, but juju is the pure antithesis of declarative config and has terrible support for baremetal and even worse for arm64.

a helm chart for a hydra operator (or even better, a full ory stack) would be huge.
